README.md

```diff
@@ -82,504 +63,41 @@ version. A copy of the GNU General Public License can be found in the
-
-When both the [DS9 image viewer](http://ds9.si.edu/site/Home.html) and
-[XPA toolset](http://hea-www.harvard.edu/RD/xpa/) are installed, the
-test suite will include tests that check that DS9 can be used from
-Sherpa. This causes several copies of the DS9 viewer to be created,
-which can be distracting, as it can cause loss of mouse focus (depending
-on how X-windows is set up). This can be avoided by installing the 
-[X virtual-frame buffer (Xvfb)](https://en.wikipedia.org/wiki/Xvfb).
-
-
-Testing Sherpa
+Using Anaconda
 --------------
 
-To test that your installation works, just type:
-
-    $ sherpa_test
-
-The number of tests run by `sherpa_test` depends on what Python
-packages are installed (for example, `astropy` and `matplotlib`), on
-external software (are DS9 and the XPA toolset installed), and whether
-the external Sherpa test data set is installed.
-
-The external Sherpa test data is large, and mostly useful when
-developing Sherpa. The download method depends on how Sherpa
-was installed.
-
-### 3a. Binary installation
-
-The external test data files can be
-installed from GitHub channel by saying:
-
-    $ pip install https://github.com/sherpa/sherpa-test-data/archive/4.10.0.tar.gz
-
-At this point, `sherpa_test` will pick up the data and so run more
-tests.
-
-### 3b. Built from source
-
-At the top level of the Sherpa distribution, used to [build
-Sherpa](#2c-build-sherpa), use the following commands to add the test
-data set into the `sherpa-test-data/` directory (this assumes that the
-source code was installed with `git` and not unpacked from a tarball):
-
-    $ git submodule init
-    $ git submodule update
-
-At this point, the data will be picked up automatically by either of
-the following commands:
-
-    $ sherpa_test
-    $ python setup.py test
-
-The data files are also available as a standard Python package with its
-own [`git` repository](https://github.com/sherpa/sherpa-test-data).
-
-
-Custom source build
-===================
-
-There are several options for customizing the Sherpa build.
-
-FFTW library
-------------
-
-Sherpa ships with the [`fftw`](http://www.fftw.org/) library source
-code and builds it as part of its own build process by
-default. However, users might want to point Sherpa to their own
-version of the library. This might be required because the performance
-of this library can be significantly increased by compiling it with
-optimization flags specific for some system or architecture.
-
-In order to make Sherpa build its modules against a local `fftw` library, users
-need to change the default Sherpa build configuration as follows.
-
-First, make sure you download the Sherpa source tarball, or get the source
-code from GitHub:
-
-    $ git clone https://github.com/sherpa/sherpa.git
-    $ cd sherpa
+Sherpa is provided for both Linux and Mac OSX operating systems running
+Python 2.7, 3.5, 3.6, and 3.7. It can be installed with by saying
```

Comment:
Is this a typo or an English construct I am unaware of?

Reply:
Can it not be both ;-)

Typo.